package.json

```diff
@@ -10,14 +10,15 @@
   },
   "homepage": "https://tutorials.ubuntu.com",
   "scripts": {
```

Comment:
Could you add a `"clean"` script which deletes at least the "node_modules/" directory?